### PR TITLE
Corrections after data folder restructure

### PR DESCRIPTION
**Pull request by lizgzil**
https://trello.com/c/ug7GeG9r/1229-clean-up-tech-grants-data

- Correct file location after files have been moved around
- A new file called data/DATA_README.md with a description of the data folder
- I also fixed https://github.com/wellcometrust/nutrition-labels/issues/80 here in [this](https://github.com/wellcometrust/nutrition-labels/pull/79/commits/82bdbb70c34ee09077beddc73c347483b9e9d33b) commit (accidentally pushed to the wrong PR, but its only a small change!)

### CODE REVIEW

**Comment by lizgzil** [comment]
[Clean up tech grants data!](https://trello.com/c/ug7GeG9r/1229-clean-up-tech-grants-data)

File: docs/Expanding_tech_grants.md [hunk 1]
```diff
@@ -52,8 +52,8 @@ An analysis showed that the model performs best when the synopsis text given is
 # 1. Retagging data points previously classified as not-tech
 
 Previously the tagged training data came from the 4 files:
-- 'data/raw/EPMC_relevant_tool_pubs_3082020.csv'
-- 'data/raw/EPMC_relevant_pubs_query2_3082020.csv'
+- 'data/raw/EPMC/EPMC_relevant_tool_pubs_3082020.csv'
```
Comment (nsorros) [inline docs/Expanding_tech_grants.md:55]: you could remove EPMC from the file names now if you want

File: representation_labels/useful_functions.py [hunk 2]
```diff
@@ -271,10 +271,10 @@ def ethnicity_tips(eth_dict):
 
 if __name__ == '__main__':
 
-    with open('data/raw/cohort_demographics_test_data.json', 'r') as fb:
+    with open('representation_labels/data/cohort_demographics_test_data.json', 'r') as fb:
         cohorts_dic = json.load(fb)
 
-    with open('data/raw/Reference_population.json', 'r') as fb:
+    with open('representation_labels/data/Reference_population.json', 'r') as fb:
```
Comment (nsorros) [inline representation_labels/useful_functions.py:277]: no more use of `data/...` convention?

File: nutrition_labels/predict.py [hunk 3]
```diff
@@ -79,12 +102,17 @@ def predict_grants(config):
...
+        default=None
+    )
 
     args = parser.parse_args()
 
     config = configparser.ConfigParser()
     config.read(args.config_path)
 
-    grant_ids, y_pred = predict_grants(config)
+    grant_ids, y_pred = predict_grants(config, args.grants_data_path)
```
Comment (nsorros) [inline nutrition_labels/predict.py:116]: why don't you add this to the config?

File: data/DATA_README.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
+# Data folder
+
+Notes about the data are given in more detail in the various docs, but here is a high level description of files and folders in the data folder.
+
+You can download this data 2 ways:
+1. If you have a Wellcome AWS account:
+`make sync_data_from_s3`
+or 
+`sync_latest_files_from_s3` for just the essential files.
```
Comment (nsorros) [inline data/DATA_README.md:9]: did you mean `make sync_latest_files_from_s3`?

**Review by nsorros — APPROVED** [review]

File: configs/clustering/2020.09.16.ini [hunk 5]
```diff
@@ -3,7 +3,7 @@ version = 2020.09.16
 description = Clustering tech grants. The clustering parameters are optimised for each tech_grants_file_name.
 
 [data]
-tech_grants_file_name = data/processed/ensemble_results.csv
+tech_grants_file_name = data/processed/ensemble/200907/ensemble_results.csv
```
Comment (aCampello) [inline configs/clustering/2020.09.16.ini:6]: Interesting that the version of the data is inside the config file. That solves some reproducibility problems.

File: configs/ensemble/2021.01.29.ini [hunk 6]
```diff
@@ -3,7 +3,7 @@ version = 2021.01.29
 description = Ensemble model on the fortytwo data
 
 [prediction_data]
-grants_data_path = data/processed/tech_grantIDs_210126_training_data_fortytwo_info.csv
+grants_data_path = data/processed/fortytwo/tech_grantIDs_210126_training_data_fortytwo_info.csv
```
Comment (aCampello) [inline configs/ensemble/2021.01.29.ini:6]: 💪

File: nutrition_labels/predict.py (hunk 3 above)
Comment (aCampello) [reply nutrition_labels/predict.py:116]: That was one of my comments in the previous PR. To enable us to predict on _any_ data, and not tie "predictions" to training, as we move along separating the functionalities.

Comment (aCampello) [reply nutrition_labels/predict.py:116]: (actually you can still add in the config file, it's just that the argument takes precedence)

**Review by aCampello — COMMENTED** [review]
Nice changes to prediction, I will check the pipeline runs for me and then accept.

**Review by aCampello — APPROVED** [review]
The test pipeline passes! :)

**Pull request merged**